job-descriptions/product-manager.md

```diff
@@ -0,0 +1,74 @@
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+You'll work with developers and other team members (from sales, marketing, and design) to plan, ship, and grow specific areas of our product. As an early member of our product team, you'll have significant product ownership and will communicate directly with our most sophisticated and demanding users to help preserve our market leadership.
+
+Why this role is special:
+
+- Our product has [extremely high engagement](https://engineeringblog.yelp.com/2019/11/winning-the-hackathon-with-sourcegraph.html) relative to most products (especially enterprise products). We get enormous amounts of user feedback from our unusually thoughtful user base.
+- Our developers use the product, so they care deeply about product matters. You won't be fighting apathy.
+
+Review our [roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit) to understand how our engineering team is organized and our [OKRs](https://about.sourcegraph.com/company/okrs) to understand the goals you will be helping drive.
+
+#### Responsibilities
+
+You will:
+
+- Own an area of the product, and create and drive its product roadmap.
+- Collaborate with engineers and designers to ideate, plan, and build the solutions to customer problems.
+- Use customer feedback to validate priorities, improve documentation, and define product requirements.
+- Be transparent in your internal and external communication. Sourcegraph is open source, and most discussion occurs in public, or in channels where you communicate directly with our customers.
+- Be a passionate advocate for a delightful user experience.
+
+#### Qualifications
+
+- Experience working as a software engineer.
+- Strong understanding of Git and Git workflows.
+- Excellent written and verbal communication skills, technical writing ability, and use of data to back up your arguments.
+- Ability to stay focused on goals and strategically prioritize work.
+- Deep passion for developer tools and productivity enhancements.
+- Demonstrated leadership, the ability to drive product strategy and work well with senior engineers and designers.
+
+#### Nice to haves
+
+- You have experience working as a product manager on a software development product.
+- You have experience building developer tools.
+- You have experience working with remote teams.
+
+### Location
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world.
+
+### Compensation and benefits
+
+We provide [competitive pay and equity](https://about.sourcegraph.com/handbook/people-ops/compensation) because we want you to act like a business owner and share in the success of Sourcegraph. We also provide [many benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.
```

Comment:
```suggestion
We provide [competitive pay and equity](https://about.sourcegraph.com/handbook/people-ops/compensation) because we want you to act like an owner and share in the success of Sourcegraph. We also provide [many benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.
```

I really like this!